manifests/Clash/ClashForWindows/0.9.11.0.yaml

```diff
@@ -0,0 +1,18 @@
+Id: fndroid.clashforwindows
```

Comment:
```suggestion
Id: Clash.ClashForWindows
```
first part of id should be as same as publisher. and the second part should be as same as the parent folder